adapter.go

```diff
@@ -613,6 +615,15 @@ func (a *Adapter) RemoveFilteredPolicy(sec string, ptype string, fieldIndex int,
 	return err
 }
 
+// checkQueryfield make sure the field won't be empty (string --> "")
+func checkQueryField(fieldValues []string) {
+	for _, fieldValue := range fieldValues {
+		if fieldValue == "" {
+			panic("the query field cannot be empty string (\"\"), please check")
```

Comment:
return error?

Reply:
ok

Reply:
My suggestion would be that instead of throwing error you just let it through in the sql
Let the consumers handle this at their end because it might be a genuine use case that I want to perform the action with empty value